tests/conftest.py

```diff
@@ -1,9 +1,21 @@
 import os
+import sys
 import pytest
 import dash
+
 from PIL import ImageFilter, ImageChops
 
 
+def _mock_impl(func):
+    return func
+
+
+# Allow everviz to be installed and have tests run without installing everest
+module = type(sys)("everest.plugins")
+module.hookimpl = _mock_impl
+sys.modules["everest.plugins"] = module
+
+
```

Comment:
I am not sure for everviz, but in the spinningjenny code this led to some tests being skipped, even if everest was installed. So I changed this to test for the presence for everest first. This may also be an issue for everviz, since there are some tests that use pytest.importorskip for test conditional on the presence of everest.

Reply:
you are right, there is a test that is being skipped but I think the test was being skipped always on travis it is not due to the change it is because we are importing the EverestPluginManager

Reply:
Yes, I guess it does not matter much for travis, but then the test is useles, right? It will never run, also not if you just want to run pytest on your own machine.